.github/workflows/optimizer.yml

```
@@ -0,0 +1,48 @@
+name: Optimize icons
+on: 
+  pull_request:
+    types:
+      - labeled
+      - synchronize
+permissions: write-all
+env:
+  SVGO: off
+jobs:
+  optimize:
+    name: Optimize icons with SVGO
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: SVGO icon optimizer
+        if: github.event.label.name == 'feature:icon'
+        uses: ericcornelissen/svgo-action@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          svgo-config: ./.github/scripts/svgo.config.js
+          svgo-version: 3
+      
+      - name: Check if there are some icon to be optimized
+        if: github.event.label.name == 'feature:icon'
+        run: echo "SVGO=on" >> $GITHUB_ENV 
+
+      - name: Get paths of added and modified files
+        uses: jitterbit/get-changed-files@v1
+        id: pr-files
+        with:
+          format: json
+
+      - name: Save paths added and modified files
+        run: echo '${{ steps.pr-files.outputs.added_modified }}' > pr-files.json
+
+      - name: Setup Python v3.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+
+      - name: Add newline
+        run: python ./.github/scripts/newliner.py ${{ env.SVGO }}
```

Any reason why we can't use the [newline-action](https://github.com/marketplace/actions/newline-action)?
```suggestion
      - name: Add newline
        uses: Logerfo/newline-action@0.0.4
        with:
          github-token: ${{ secrets.GITHUB_TOKEN }}
```

This would add newline to any file, not just SVG files. 
It's also possible to ignore files in a config (see link above)

Come to think of it, maybe it would be better to add the newline in a separate script like this:

```yml
name: Add newline
on:
  pull_request:
    types: [synchronize, opened]
jobs:
  build:
    name: newline-action
    runs-on: ubuntu-16.04
    steps:
      - uses: actions/checkout@master
      - uses: Logerfo/newline-action@0.0.4
        with:
          github-token: ${{ secrets.GITHUB_TOKEN }} # The `GITHUB_TOKEN` secret.
          config-path: .github/newline.yml # The path of the addtional configurations file.
```

---

This sounds good to me. 👍